### PR TITLE
fix: use built-in slices package from go1.21 for simplify code

### PR DESCRIPTION
**Pull request by lx1036**
use go(>=1.21) has slices package to simplify code.

### CODE REVIEW

File: pkg/controllers/podgroup/pg_controller.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog/v2"
+	"slices"
```
Comment (lowang-bh) [inline pkg/controllers/podgroup/pg_controller.go:30]: Please sort the imports.

Comment (lx1036) [reply pkg/controllers/podgroup/pg_controller.go]: fixed

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Comment by hzxuzhonghu** [comment]
BTW, should we remove the `Contains` from util pkg?

**Comment by hzxuzhonghu** [comment]
@Monokaix The repo setting is not quite correct, without running the CI jobs, it got merged

**Comment by Monokaix** [comment]
> @Monokaix The repo setting is not quite correct, without running the CI jobs, it got merged

I will check that！

**Pull request merged**